src/services/paymentService.js

```diff
@@ -0,0 +1,19 @@
+const paymentDao = require("../models/paymentDao");
+const axios = require("axios");
+
+const createPaymentHistory = async (orderId, paymentKey, amount) => {
+  return await paymentDao.createPaymentHistory(orderId, paymentKey, amount);
+};
+const getPaymentHistory = async () => {
+  console.log(process.env.SECRET_KEY);
+  return await axios.get("https://api.tosspayments.com/v1/payments/orders/QIZBXKZ_q7aV5k63nFbOt", {
+    headers: {
+      Authorization: `Basic ${process.env.SECRET_KEY}`,
+    },
+  });
+};
```

Comment:
```suggestion
const getPaymentHistory = async () => {
  const response = await axios.get("https://api.tosspayments.com/v1/payments/orders/QIZBXKZ_q7aV5k63nFbOt", {
    headers: {
      Authorization: `Basic ${process.env.SECRET_KEY}`,
    },
  });
  
  return response.data;
};
```

위 피드백에 이어지는 내용입니다. 애초에 .data를 return 해주는 내용이에요